docs/_templates/searchbox.html

```diff
@@ -0,0 +1,6 @@
+<div role="search">
+  <form id="rtd-search-form" class="wy-form" action="{{ pathto('search') }}" method="get">
```

Comment:
The search role isn’t valid on a `div`. Hopefully no styles are attached to this? If possible we should also remove the unneeded div.

```suggestion
<div>
  <form id="rtd-search-form" class="wy-form" action="{{ pathto('search') }}" method="get" role="search">
```

Reply:
Seems like it works without the `div` and the role moved.